internal/cache/valuedescriptors.go

```diff
@@ -9,6 +9,7 @@ package cache
 
 import (
 	"fmt"
+
```

Comment:
Again, this is just whitespace cleanup... Either put all of your whitespace changes into a single commit, or as with the const changes, a separate PR is more appropriate.

Reply:
will do